examples/slack-connector/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, salesforce.com, inc.
+  ~ All rights reserved.
+  ~ SPDX-License-Identifier: BSD-3-Clause
+  ~ For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.salesforce.chatbot</groupId>
+  <artifactId>slack-demo-app-connector</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
version should be same as parent, refer:
https://github.com/forcedotcom/einstein-bot-channel-connector/blob/dabfbe210bace12680ab1a4249f54d493b2535a5/examples/twitter-connector/pom.xml#L13